app/main/model/image.py

```diff
@@ -22,14 +25,13 @@ class ImageModel(db.Model):
   doc_id = db.Column(db.String(64, convert_unicode=True), nullable=True, index=True, unique=True)
   phash = db.Column(db.BigInteger, nullable=True, index=True)
   pdq = db.Column(BIT(256), nullable=True, index=True)
-
+  sscd = db.Column(Vector(512), nullable=True, index=True)
```

Comment:
Let's remove `index=True` here @ahmednasserswe . We'll want to stress test on QA and later will likely add an approximate nearest neighbor using HNSW (https://tembo.io/blog/vector-indexes-in-pgvector/)